stream-ingest/src/main/java/com/dellemc/oe/ingest/EventWithTimestampWriter.java

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
It seems that EvnetWithTimestampWriter, EventWriter, and JSONWriter do nearly the same thing. I recommend consolidating to a single class.